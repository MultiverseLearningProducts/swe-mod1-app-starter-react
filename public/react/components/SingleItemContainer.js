import React, {useState,useEffect} from 'react'
import {useParams,Link} from "react-router-dom";
import { Item } from './Item';
import {Cart} from './Cart';
import Button from '@mui/material/Button';
import ButtonGroup from '@mui/material/ButtonGroup';
import Rating from '@mui/material/Rating';
import FacebookIcon from '@mui/icons-material/Facebook';
import TwitterIcon from '@mui/icons-material/Twitter';
import InstagramIcon from '@mui/icons-material/Instagram';

export default function SingleItemContainer(props) {
    const [singleItem ,setSingleItem]=useState({ });
    let { id } = useParams();
    const { onAdd } = props;

    async function fetchSingleProduct(id) {
        try {
        const response = await fetch(`http://localhost:3000/products/${id}`);
        const responseJSON = await response.json();
        console.log (responseJSON.product);
    
        setSingleItem(responseJSON.product);
    
        } catch (err) {
        console.log("Oh no an error! ", err);
        }
    }

    useEffect(() => {
        fetchSingleProduct(id);
    }, []);


return (
    
    <div className="single-item-container">
      <section className="singleitem-left" aria-label="shopping tabs">
      <div className="single-item">
          <img src={singleItem.image} alt={singleItem.name} className="singleItem-img"/><br/>
      </div>
      </section>
      <section className="singleitem-center" aria-label="home tab">
        <div className="plant-image">
          <img src={singleItem.image} alt={singleItem.name} className="item-img"/><br/>
        </div>
        <div className="plant-image">
          <img src={singleItem.image} alt={singleItem.name} className="item-img"/><br/>
        </div>
      </section>
      <section className="singleitem-right" aria-label="cart tab">
      <div className="single-item">
      <Rating name="size-medium" defaultValue={2} />
        <h2 className="card-title">{singleItem.name}</h2>
          <h3 className="card-title"> ${singleItem.price} USD</h3>
          <h3 >{singleItem.description}</h3> 
          <div className="button-group">
      <ButtonGroup variant="contained" aria-label="outlined primary button group">
        <Button component={Link} 
        // <If condition={props.item.type=='plants'}>
        // </If>
        to={'/plants'}>Back</Button>
        <Button className="cart-btn"
            aria-label="Add To Cart"
            onClick={() => onAdd(singleItem)}>Add to Cart</Button>
    
    </ButtonGroup>
    <h5>CATEGORIES:<span>Home,{singleItem.type}</span></h5>
      <h5>SHARE <span>    </span>
      <span>
      <FacebookIcon></FacebookIcon>
      <TwitterIcon></TwitterIcon>
      <InstagramIcon></InstagramIcon>
      </span>
      </h5>

      </div>
      </div>
      </section>
    </div>

 )
}
