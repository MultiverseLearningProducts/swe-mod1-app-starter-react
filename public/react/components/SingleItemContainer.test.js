import React from "react";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import SingleItemContainer from "./SingleItemContainer";

describe("Single Item container component", () => {
  test("renders Single Item Container component as expected", () => {
    const plant = {
      name: "Fan Flower",
      image:
        "https://cdn.shopify.com/s/files/1/0317/0687/3992/products/191-large_default_bdf68ca7-5284-412c-9fec-78235d9cd046.jpg?v=1580438028",
      price: "12.99",
      type: "plant",
      description:
        "Fusce fermentum odio nec arcu. Praesent vestibulum dapibus nibh. Nullam nulla eros, ultricies sit amet, nonummy id, imperdiet feugiat, pede. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Phasellus dolor",
    };
    const component = renderer.create(
      <MemoryRouter initialEntries={["/plant/1"]}>
        <SingleItemContainer item={plant} />
      </MemoryRouter>
    );
    // console.log("What does our snapshot look like?  ", snapshot);
    const snapshot = component.toJSON();
    expect(snapshot).toMatchSnapshot();
  });

  // test("Invokes addItem function when clicked", () => {
  //   const plant = {
  //     name: "Fan Flower",
  //     image:
  //       "https://cdn.shopify.com/s/files/1/0317/0687/3992/products/191-large_default_bdf68ca7-5284-412c-9fec-78235d9cd046.jpg?v=1580438028",
  //     price: "12.99",
  //     type: "plant",
  //     description:
  //       "Fusce fermentum odio nec arcu. Praesent vestibulum dapibus nibh. Nullam nulla eros, ultricies sit amet, nonummy id, imperdiet feugiat, pede. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Phasellus dolor",
  //   };
  //   const stub = () => "clicked!!!";
  //   //Mock item
  //   const component = renderer.create(
  //     <MemoryRouter initialEntries={["/plant/1"]}>
  //       <SingleItemContainer item={plant} onAdd={stub} />
  //     </MemoryRouter>
  //   );
  //   //Let's unspool the tree
  //   const treeEl = component.toTree();
  //   //console.log("What does our tree look like?", treeEl);
  //   expect(treeEl.props.addItem()).toBe("clicked!!!");
  // });
});
