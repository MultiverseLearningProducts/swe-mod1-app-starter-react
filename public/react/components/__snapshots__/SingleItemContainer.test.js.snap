// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single Item container component renders Single Item Container component as expected 1`] = `
<div
  className="single-item-container"
>
  <section
    aria-label="shopping tabs"
    className="singleitem-left"
  >
    <div
      className="single-item"
    >
      <img
        className="singleItem-img"
      />
      <br />
    </div>
  </section>
  <section
    aria-label="home tab"
    className="singleitem-center"
  >
    <div
      className="plant-image"
    >
      <img
        className="item-img"
      />
      <br />
    </div>
    <div
      className="plant-image"
    >
      <img
        className="item-img"
      />
      <br />
    </div>
  </section>
  <section
    aria-label="cart tab"
    className="singleitem-right"
  >
    <div
      className="single-item"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-iljtu2-MuiRating-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;position:relative;font-size:1.5rem;color:#faaf00;cursor:pointer;text-align:left;-webkit-tap-highlight-color:transparent;}.css-iljtu2-MuiRating-root.Mui-disabled{opacity:0.38;pointer-events:none;}.css-iljtu2-MuiRating-root.Mui-focusVisible .MuiRating-iconActive{outline:1px solid #999;}.css-iljtu2-MuiRating-root .MuiRating-visuallyHidden{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;white-space:nowrap;width:1px;}",
          }
        }
        data-emotion="css iljtu2-MuiRating-root"
      />
      <span
        aria-label={null}
        className="MuiRating-root MuiRating-sizeMedium css-iljtu2-MuiRating-root"
        onMouseLeave={[Function]}
        onMouseMove={[Function]}
        role={null}
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-dqr9h-MuiRating-label{cursor:inherit;}",
            }
          }
          data-emotion="css dqr9h-MuiRating-label"
        />
        <label
          className="css-dqr9h-MuiRating-label"
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-7qmtgc-MuiRating-icon{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-transition:-webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;pointer-events:none;}",
              }
            }
            data-emotion="css 7qmtgc-MuiRating-icon"
          />
          <span
            className="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
          >
            <style
              dangerouslySetInnerHTML={
                Object {
                  "__html": ".css-1vooibu-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:inherit;}",
                }
              }
              data-emotion="css 1vooibu-MuiSvgIcon-root"
            />
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="StarIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
              />
            </svg>
          </span>
          <span
            className="MuiRating-visuallyHidden"
          >
            1 Star
          </span>
        </label>
        <input
          checked={false}
          className="MuiRating-visuallyHidden"
          disabled={false}
          name="size-medium"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          type="radio"
          value={1}
        />
        <label
          className="css-dqr9h-MuiRating-label"
        >
          <span
            className="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="StarIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
              />
            </svg>
          </span>
          <span
            className="MuiRating-visuallyHidden"
          >
            2 Stars
          </span>
        </label>
        <input
          checked={true}
          className="MuiRating-visuallyHidden"
          disabled={false}
          name="size-medium"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          type="radio"
          value={2}
        />
        <label
          className="css-dqr9h-MuiRating-label"
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-1c99szj-MuiRating-icon{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-transition:-webkit-transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;pointer-events:none;color:rgba(0, 0, 0, 0.26);}",
              }
            }
            data-emotion="css 1c99szj-MuiRating-icon"
          />
          <span
            className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="StarBorderIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
              />
            </svg>
          </span>
          <span
            className="MuiRating-visuallyHidden"
          >
            3 Stars
          </span>
        </label>
        <input
          checked={false}
          className="MuiRating-visuallyHidden"
          disabled={false}
          name="size-medium"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          type="radio"
          value={3}
        />
        <label
          className="css-dqr9h-MuiRating-label"
        >
          <span
            className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="StarBorderIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
              />
            </svg>
          </span>
          <span
            className="MuiRating-visuallyHidden"
          >
            4 Stars
          </span>
        </label>
        <input
          checked={false}
          className="MuiRating-visuallyHidden"
          disabled={false}
          name="size-medium"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          type="radio"
          value={4}
        />
        <label
          className="css-dqr9h-MuiRating-label"
        >
          <span
            className="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="StarBorderIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z"
              />
            </svg>
          </span>
          <span
            className="MuiRating-visuallyHidden"
          >
            5 Stars
          </span>
        </label>
        <input
          checked={false}
          className="MuiRating-visuallyHidden"
          disabled={false}
          name="size-medium"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          type="radio"
          value={5}
        />
        <label
          className="MuiRating-label MuiRating-pristine css-dqr9h-MuiRating-label"
        >
          <input
            checked={false}
            className="MuiRating-visuallyHidden"
            id="size-medium-empty"
            name="size-medium"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            type="radio"
            value=""
          />
          <span
            className="MuiRating-visuallyHidden"
          >
            Empty
          </span>
        </label>
      </span>
      <h2
        id="name-title"
      />
      <h3
        className="card-title"
      >
         $
         USD
      </h3>
      <h3
        id="description-title"
      />
      <div
        className="button-group"
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-dr3dyz-MuiButtonGroup-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;border-radius:4px;box-shadow:0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);}.css-dr3dyz-MuiButtonGroup-root .MuiButtonGroup-grouped{min-width:40px;box-shadow:none;}.css-dr3dyz-MuiButtonGroup-root .MuiButtonGroup-grouped:not(:first-of-type){border-top-left-radius:0;border-bottom-left-radius:0;}.css-dr3dyz-MuiButtonGroup-root .MuiButtonGroup-grouped:not(:last-of-type){border-top-right-radius:0;border-bottom-right-radius:0;border-right:1px solid #bdbdbd;border-color:#1565c0;}.css-dr3dyz-MuiButtonGroup-root .MuiButtonGroup-grouped:not(:last-of-type).Mui-disabled{border-right:1px solid rgba(0, 0, 0, 0.26);}.css-dr3dyz-MuiButtonGroup-root .MuiButtonGroup-grouped:hover{box-shadow:none;}",
            }
          }
          data-emotion="css dr3dyz-MuiButtonGroup-root"
        />
        <div
          aria-label="outlined primary button group"
          className="MuiButtonGroup-root MuiButtonGroup-contained css-dr3dyz-MuiButtonGroup-root"
          role="group"
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-1yu95zv-MuiButton-root{font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 16px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#fff;background-color:#1976d2;box-shadow:0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:#1565c0;box-shadow:0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);}@media (hover: none){.css-1yu95zv-MuiButton-root:hover{background-color:#1976d2;}}.css-1yu95zv-MuiButton-root:active{box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root.Mui-focusVisible{box-shadow:0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);box-shadow:none;background-color:rgba(0, 0, 0, 0.12);}",
              }
            }
            data-emotion="css 1yu95zv-MuiButton-root"
          />
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": ".css-sghohy-MuiButtonBase-root-MuiButton-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 16px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#fff;background-color:#1976d2;box-shadow:0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root::-moz-focus-inner{border-style:none;}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-sghohy-MuiButtonBase-root-MuiButton-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-sghohy-MuiButtonBase-root-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:#1565c0;box-shadow:0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);}@media (hover: none){.css-sghohy-MuiButtonBase-root-MuiButton-root:hover{background-color:#1976d2;}}.css-sghohy-MuiButtonBase-root-MuiButton-root:active{box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-focusVisible{box-shadow:0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);box-shadow:none;background-color:rgba(0, 0, 0, 0.12);}",
              }
            }
            data-emotion="css sghohy-MuiButtonBase-root-MuiButton-root"
          />
          <a
            className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContainedPrimary css-sghohy-MuiButtonBase-root-MuiButton-root"
            href="/coffees"
            id="back-btn"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
          >
            Back
          </a>
          <button
            aria-label="Add To Cart"
            className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedContained MuiButtonGroup-groupedContainedHorizontal MuiButtonGroup-groupedContainedPrimary css-sghohy-MuiButtonBase-root-MuiButton-root"
            disabled={false}
            id="add-btn"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            Add to Cart
          </button>
        </div>
        <h5>
          CATEGORIES:
          <span>
            Home,
          </span>
        </h5>
        <h5>
          SHARE 
          <span>
                
          </span>
          <span>
            <style
              dangerouslySetInnerHTML={
                Object {
                  "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
                }
              }
              data-emotion="css i4bv87-MuiSvgIcon-root"
            />
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="FacebookIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
              />
            </svg>
            <style
              dangerouslySetInnerHTML={
                Object {
                  "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
                }
              }
              data-emotion="css i4bv87-MuiSvgIcon-root"
            />
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="TwitterIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
              />
            </svg>
            <style
              dangerouslySetInnerHTML={
                Object {
                  "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
                }
              }
              data-emotion="css i4bv87-MuiSvgIcon-root"
            />
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="InstagramIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
              />
            </svg>
          </span>
        </h5>
      </div>
    </div>
  </section>
</div>
`;
